### DISCUSSION TITLE
Dashboard widget device filtering

### DISCUSSION THREAD

**Discussion by gdprdatasubect** [netbox-community/netbox#12530] [Q&A] [answered]
Hello everyone, i kinda expected the filters for the Dashboard-Widgets to just work by copy-pasting the Code from saved filters.

This is apparently not the case, e.g. 
```
{
    "manufacturer_id": [
        "3"
    ],
    "status": [
        "active"
    ]
}
```

Produces the error "Invalid filter specification for devices. ", but it works fine as saved filter.

What am i doing wrong?

**Comment by kkthxbye-code** [comment]
The filter field for the Object Count widget is a QuerySet filter. The Object List widget and Saved Filters are URL parameters. Saved Filters and Object List widget are compatible with each other, the Object Count widget is not.

**Comment by abhi1693** [comment, marked as answer]
Try this instead, it should be ORM filter query

```json
{"status": ["active"], "device_type__manufacturer_id": "3"}
```

**Reply by gdprdatasubect** [reply]
Thanks to you and @kkthxbye-code, that helped me quite a lot.
Should i create an issue and try to improve the documentation?
